graphix/pauli.py

```diff
@@ -284,6 +338,32 @@ def parse(name: str) -> Pauli:
     return get(IXYZ[name], UNIT)
 
 
+def is_int(value: Number) -> bool:
+    return value == int(value)
+
+
+class Measure(typing.NamedTuple):
+    """
+    Pauli measurement.
+    """
+
+    axis: Axis
+    sign: Sign
+
+    @staticmethod
+    def try_from(plane: Plane, angle: float) -> Measure | None:
+        angle_double = 2 * angle
+        if not is_int(angle_double):
+            return None
+        angle_double_mod_4 = int(angle_double) % 4
+        if angle_double_mod_4 % 2 == 0:
+            axis = plane.cos
+        else:
+            axis = plane.sin
+        sign = Sign.minus_if(angle_double_mod_4 >= 2)
+        return Measure(axis, sign)
```

Comment:
perhaps differnet class name is good to make clear it's specifically to work with Pauli measurement?

Reply:
Fixed in e374cd5.

Reply:
This definitely indicates that unification is needed! However, I don’t think this necessarily dictates a specific order for merging:
- Regardless of which PR is merged first, the others can be unified during the rebasing process.
- It seems to me that the primary need for unification is between `MeasurementDescription` (in #178) and `Measurement` (in #191), as both classes are intended for general measurement (and neither of these PRs introduces a class specifically for Pauli measures).
- `pauli.py` seems like an appropriate place for defining `PauliMeasure`.

Reply:
sorry - could you rename to `PauliMeasurement` to be consistent with other classes mentioned above?

Reply:
Fixed in 6388d4.